### ISSUE TITLE
refactor input parser

### ISSUE THREAD

**Issue by koomie** [pecos/tps#87]

**Comment by koomie** [comment]
Updated parsed landed with #88.  

An example of the updated ini-style input is available [here](https://github.com/pecos/tps/blob/main/src/runfileExample.ini).

**Comment by koomie** [comment]
A companion python utility which endeavors to update a previous-style input is available at: `utils/update_input_file.py`.  To use, supply a path to the desired input file to convert via command-line option:

```
$ ./utils/update_input_file.py
usage: update_input_file.py [-h] --infile INFILE
```

The converted file will be written to stdout. If an input parameter is provided that is unknown, the conversion utility should fail.
